packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1137,7 +1137,7 @@ The first three parameters in the table below correspond to different types of l
 
```

Comment:
I'd recommend re-ordering 
from:
```
- [List All Users](#list-all-users) (no parameters)
- [Find Users](#find-users) (`q`)
- [List Users with a Filter](#list-users-with-a-filter) (`filter`)
- [List Users with Search](#list-users-with-search) (`search`)
```
to:
```
- [List Users with Search](#list-users-with-search) (`search`)
- [Find Users](#find-users) (`q`)
- [List Users with a Filter](#list-users-with-a-filter) (`filter`)
- [List All Users](#list-all-users) (no parameters)
```

Reply:
Reordered.